_episodes/04-sharing-environments.md

```diff
@@ -189,7 +189,7 @@ make sure to add the `--from-history` argument to the `conda env export` command
 > > ~~~
 > > $ mkdir scikit-learn-project-dir
 > > $ cd scikit-learn-project-dir
-> > $ nano environment.yml
+> > $ nano environment.yml  # Adding the above contents to this file, saving and exiting.

```

Comment:
Hmm. Nano won't work if it's not installed. One to put a pin in for later, perhaps?